_data/contents.yml

```diff
@@ -37,6 +37,7 @@ specifics:
   - ets
   - mnesia
   - debugging
+  - benchmarking
```

Comment:
Since this is really a lesson about using `Benchee`, I think this might be better suited for the `Libraries` category and a new title:  "Benchmarking with Benchee".

I may even be worth mentioning https://github.com/alco/benchfella which could be argued is more popular (used by core members, greater number of stars), just so folks are aware there _are_ others.

Reply:
Not to start a popularity contest, but benchee is also used by core members, number of stars is almost identical and benchees dl numbers are growing (finally reached ~half of weekly from benchfella :tada: ) while benchfella is mostly in maintenance. Most of DL/star ahead comes from it just being around almost thrice as long imo.

Not against moving it to libraries, just sayin ;P